### PR TITLE
make CSS http://jigsaw.w3.org/css-validator/ compliant.

### PR DESCRIPTION
**Pull request by mro**
see http://jigsaw.w3.org/css-validator/validator?uri=https://rawgithub.com/mro/Shaarli/hotfix/css/inc/reset.css
and http://jigsaw.w3.org/css-validator/validator?uri=https://rawgithub.com/mro/Shaarli/hotfix/css/inc/shaarli.css

### CODE REVIEW

File: inc/shaarli.css [hunk 1]
```diff
@@ -740,7 +740,7 @@ a.qrcode img {
     background-color: transparent;
     background-color: rgba(0, 0, 0, 0.4);
     /* FF3+, Saf3+, Opera 10.10+, Chrome, IE9 */
-    filter: progid: DXImageTransform.Microsoft.gradient(startColorstr=#66000000, endColorstr=#66000000);
+    /* filter: progid:DXImageTransform.Microsoft.gradient(startColorstr=#66000000, endColorstr=#66000000); */
```
Comment (ArthurHoaro) [inline inc/shaarli.css]: You probably can remove this.

**Comment by ArthurHoaro** [comment]
:+1:

File: inc/shaarli.css (hunk 1 above)
Comment (mro) [inline inc/shaarli.css]: yes, I think so, too. But I have no clue what it does, so I left purging to the more confident ones.

Comment (ArthurHoaro) [inline inc/shaarli.css]: A quick google search told me it's probably a hack for IE9. You can remove it. :)

**Comment by nicolasdanelon** [comment]
good job  @mro  !

**Comment by ArthurHoaro** [comment]
@mro Could you remove the commented line to merge your PR?

**Comment by mro** [comment]
no

**Comment by ArthurHoaro** [comment]
Hm... any particular reason?

**Comment by mro** [comment]
no particular one. Only what I mentioned in https://github.com/shaarli/Shaarli/pull/476#discussion_r52880653 plus that I won't rework each PR. Either take it or leave it. No hard feelings.

**Pull request closed without merging**